source/install.sh

```diff
@@ -2,13 +2,54 @@
 SCRIPT=$(readlink -f "${BASH_SOURCE[0]}")
 SOURCE_PATH=$(dirname "$SCRIPT")
 
-# options
-# TODO: parse arguments and provide --help to set these options
 BUILD_TESTING="OFF"
 BUILD_CONTROLLERS="ON"
 BUILD_DYNAMICAL_SYSTEMS="ON"
 BUILD_ROBOT_MODEL="ON"
 
+FAIL_MESSAGE="The provided input arguments are not valid.
+Run the script with the '-h' or '--help' argument."
+
+HELP_MESSAGE="Usage: ./install.sh [OPTIONS]
+
+An install script for the control libraries.
+
+Options:
+  -c, --controllers        Build the controllers library ('ON'|'OFF')
+  -d, --dynamical_systems  Build the dynamical systems library ('ON'|'OFF')
+  -r, --robot_model        Build the robot model library ('ON'|'OFF')
+  -t, --testing            Build and run the unit tests ('ON'|'OFF')
```

Comment:
This doesn't actually run the tests, it just builds the test targets (it doesn't even "install" the tests technically, they just live in the build directory)

Reply:
ah yes good idea, I might go for the `no-controllers` solution, that will also simplify the parsing of the arguments